### DISCUSSION TITLE
Sending telemetry to Azure Data Explorer

### DISCUSSION THREAD

**Discussion by tashivani** [open-telemetry/opentelemetry-cpp#2176] [Q&A]
Hi there,
I have been trying to send telemetry from a C++ desktop app to Azure (Azure Data Explorer). 
I setup a project that sends telemetry using grpc exporter to a collector running on the same machine. The collector is able to write the telemetry to local files but when I try sending it to Azure, I don't see any data in the ADX tables I created. I also don't see any error messages on the collector side. I have followed the documentation provided in this link so far to get Azure setup - [Create an Azure Active Directory application registration in Azure Data Explorer](https://learn.microsoft.com/en-us/azure/data-explorer/provision-azure-ad-app). I have also attached the otel collector yml file here for reference in case I'm making a mistake in the yaml file. I'm running out of ideas on how to debug this so any help would be appreciated. If anyone has a working example to share, that would also be great.  Thank you :)
[otel-collector.yml.md](https://github.com/open-telemetry/opentelemetry-collector-contrib/files/11637202/otel-collector.yml.md)
![image](https://github.com/open-telemetry/opentelemetry-cpp/assets/10983792/214a7ebe-e282-44aa-960f-0c0e2a41a232)

**Comment by lalitb** [comment]
I can see you have also raised this question at opentelemetry-collector-contrib repo - https://github.com/open-telemetry/opentelemetry-collector-contrib/discussions/23038. That is the right place to ask the question, as this is not directly related to C++ SDK. You can also post this question at Stackoverflow, tagging `open-telemetry`, `open-telemetry-collector`, `azure-data-explorer`.

**Reply by tashivani** [reply]
Thank you! I have closed this discussion and I will try posting on Stackoverflow as well with the tag you mentioned.
